### PR TITLE
remove token

### PR DESCRIPTION
**Pull request by azime**

### CODE REVIEW

**Review by kadhikari — APPROVED** [review]

**Review by pbench — APPROVED** [review]

File: artemis/common_fixture.py [hunk 1]
```diff
@@ -36,11 +36,11 @@ def clean_kirin_db():
         conn.commit()
 
         cur.execute(
-            "INSERT INTO contributor SELECT 'realtime.sherbrooke','ca-qc-sherbrooke','token_to_be_modified',"
+            "INSERT INTO contributor SELECT 'realtime.sherbrooke','ca-qc-sherbrooke','',"
```
Comment (woshilapin) [inline artemis/common_fixture.py:39]: Do you want to keep this empty string?

```suggestion
            "INSERT INTO contributor SELECT 'realtime.sherbrooke','ca-qc-sherbrooke',"
```

File: artemis/common_fixture.py [hunk 2]
```diff
@@ -36,11 +36,11 @@ def clean_kirin_db():
...
             "'feed_url_to_be_modified','gtfs-rt'"
         )
         cur.execute(
-            "INSERT INTO contributor SELECT 'realtime.cots','guichet-unique','token_to_be_modified',"
+            "INSERT INTO contributor SELECT 'realtime.cots','guichet-unique','',"
```
Comment (woshilapin) [inline artemis/common_fixture.py:43]: Same here?

```suggestion
            "INSERT INTO contributor SELECT 'realtime.cots','guichet-unique',"
```

Comment (azime) [reply artemis/common_fixture.py]: Done

File: artemis/common_fixture.py (hunk 1 above)
Comment (woshilapin) [reply artemis/common_fixture.py]: I actually realize that I didn't fully understand what my comment meant (it's an `INSERT`, not a list of random string). Anyway, thanks, the new code is nicer anyway :+1:

**Review by woshilapin — APPROVED** [review]

**Pull request merged**